src/Manager.java

```diff
@@ -0,0 +1,81 @@
+
+public class Manager {
+	
+	public static void main(String args[])
+	{
+		int turns = 1;
+		
+		boolean isCaught = false;
+		Hare hare = new Hare(0,0);
+		Hound hound = new Hound(0,0);
+		hound.setDirection(hare.getDirection());
+		while(!isCaught && turns <= 100)
+		{
+			System.out.println("Turn:" +turns);
+			if(turns <=3)
+			{
+				hare.move(3, hare.getDirection());
+			}
+			else 
+			{
+				if(Math.sqrt(Math.pow(getXDisplacement(hare,hound), 2) + Math.pow(getYDisplacement(hare,hound), 2)) <=2)
+				{
+					isCaught = true;
+					break;
+				}
+					
+				if(turns % 3 == 0)
+				{
+					hare.setDirection(Directions.randomDirection());
+				}
+				if(turns % 5 == 0)
+				{
+					if(getXDisplacement(hare,hound) > 0 && getYDisplacement(hare,hound) > 0 && getXDisplacement(hare,hound) > getYDisplacement(hare,hound) )
+					   hound.setDirection(Directions.SOUTH);
+					else if(getXDisplacement(hare,hound) > 0 && getYDisplacement(hare,hound) > 0 && getXDisplacement(hare,hound) < getYDisplacement(hare,hound) )
+					   hound.setDirection(Directions.WEST);
+					else if(getXDisplacement(hare,hound) < 0 && getYDisplacement(hare,hound) < 0 && getXDisplacement(hare,hound) < getYDisplacement(hare,hound) )
+					   hound.setDirection(Directions.NORTH);
+					else if(getXDisplacement(hare,hound) < 0 && getYDisplacement(hare,hound) < 0 && getXDisplacement(hare,hound) > getYDisplacement(hare,hound) )
+					   hound.setDirection(Directions.EAST);
+					else if(getXDisplacement(hare,hound) < 0 && getYDisplacement(hare,hound) > 0 && Math.abs(getXDisplacement(hare,hound)) < getYDisplacement(hare,hound) )
+						   hound.setDirection(Directions.WEST);
+					else if(getXDisplacement(hare,hound) < 0 && getYDisplacement(hare,hound) > 0 && Math.abs(getXDisplacement(hare,hound)) > getYDisplacement(hare,hound) )
+						   hound.setDirection(Directions.NORTH);
+					else if(getXDisplacement(hare,hound) > 0 && getYDisplacement(hare,hound) < 0 && getXDisplacement(hare,hound) > Math.abs(getYDisplacement(hare,hound)) )
+						   hound.setDirection(Directions.SOUTH);
+					else 
+						hound.setDirection(Directions.EAST);
+					
+					
+				}
+				
+				hare.move(3, hare.getDirection());
+				if(Math.abs(getXDisplacement(hare,hound)) > Math.abs(getYDisplacement(hare,hound)) )
+				  hound.move(Math.abs(getYDisplacement(hare,hound)) - Math.abs(Math.abs(getYDisplacement(hare,hound)) - 8) , hound.getDirection());
```

Comment:
The logic in this line above makes no sense as far as I can tell. I think what you wanted to do was move by either Math.abs(getYDisplacement(hare,hound)) or by 8, whichever is smaller, which you can do with the "Math.min" method. Same deal with the else branch.